chain-impl-mockchain/src/leadership/genesis.rs

```diff
@@ -58,7 +58,7 @@ impl GenesisLeaderSelection {
 
     pub fn leader(
         &self,
-        vrf_key: &SecretKey<Curve25519_2HashDH>,
+        _vrf_key: &SecretKey<Curve25519_2HashDH>,
```

Comment:
I agree that they those are annoying but shutting the warning when it's a WIP is not a good idea. In this case it also conflict with my next PR